### ISSUE TITLE
Quotes in s/// seem to cause problems

### ISSUE THREAD

**Issue by Odud** [MadcapJake/language-perl6fe#14]
Please see attached image - it seems as though the quote character is causing the highlighter to think that everything that follows is a literal 
![capture](https://cloud.githubusercontent.com/assets/15649511/12216298/7b189668-b6d4-11e5-8cfa-568dcf7b6fed.PNG)
string

**Comment by MadcapJake** [comment]
Triple slash constructs like `s///` and `tr///` aren't really supported yet.  I am not really sure how to crack this nut.  The problem is that Atom's highlighter doesn't support multi-part forms.

Let me at least add single-line support and I have a few ideas for how I could try to fake multi-part constructs.

**Comment by Odud** [comment]
It's not a big deal as it doesn't happen that often, at the moment I just put stuff like this at the end of the file... But of course a fix would be great.

**Comment by Odud** [comment]
Hi Jake, off topic but is it possible to have perl and perl6fe working in atom at the same time? At the moment unless I disable perl I cant get seem to get perl6fe to work. Do you have an example config where both work?

Pete Barlow

-----Original Message-----

**Comment by MadcapJake** [comment]
Yeah, just follow the third option in [How Do I Use This?](https://github.com/MadcapJake/language-perl6fe#how-do-i-use-this) and only include filenames that are perl6-specific.  If you ever find a perl 6 file that ends in `pl` or `pm`, you'll have to just use the language selector (`ctrl+shift+l`).

**Comment by Odud** [comment]
Thanks, I think that's what I'd done, probably messed something up in the config, can't test atm as I'm at the pub...

Pete Barlow

-----Original Message-----
